classes/UpgradeTools/Translator.php

```diff
@@ -51,19 +61,26 @@ public function __construct($caller)
      */
     public function trans($id, array $parameters = [], $domain = 'Modules.Autoupgrade.Admin', $locale = null)
     {
-        // If PrestaShop core is not instancied properly, do not try to translate
+        // Get PrestaShop version
+        $psVersion = _PS_VERSION_;
+
+        // If PrestaShop core is not instantiated properly, do not try to translate
         if (!method_exists('\Context', 'getContext') || null === \Context::getContext()->language) {
             return $this->applyParameters($id, $parameters);
         }
 
-        if (method_exists('\Translate', 'getModuleTranslation')) {
-            $translated = \Translate::getModuleTranslation('autoupgrade', $id, $this->caller, null);
-            if (!count($parameters)) {
-                return $translated;
-            }
-        } else {
-            $translated = $id;
+        // Use new translation system for PrestaShop 1.7.6 and newer
+        if (version_compare($psVersion, '1.7.6.0', '>=')) {
```

Comment:
I did not test this guys, but I think that you need to targe 1.7.8 instead. 🤔

1.7.8 is the first version of Prestashop, that could load translations from `modules/modulename/translations` folder, on 1.7.6 and 1.7.7, they need to be imported manually into database.